crates/core/src/contract/executor.rs

```diff
@@ -547,6 +547,28 @@ pub(crate) trait ContractExecutor: Send + 'static {
 /// A WASM executor which will run any contracts, delegates, etc. registered.
 ///
 /// This executor will monitor the store directories and databases to detect state changes.
+/// Represents an operation that's waiting for a contract to finish initializing
+#[derive(Debug)]
+#[allow(dead_code)] // Fields will be used when we implement proper queue processing
```

Comment:
why do we have dead_code here, remove

Reply:
Fixed! Removed the #[allow(dead_code)] annotation. The fields are now actively used in logging when operations are queued.